File: src/main/java/no/nav/foreldrepenger/boot/conditionals/OnFSSCondition.java
package no.nav.foreldrepenger.boot.conditionals;

import static no.nav.foreldrepenger.boot.conditionals.Cluster.fssClusters;

import org.springframework.core.type.AnnotatedTypeMetadata;

public class OnFSSCondition extends OnClusterCondition {

    @Override
    protected Cluster[] clusters(AnnotatedTypeMetadata md) {
        return fssClusters();
    }
}

File: src/main/kotlin/no/nav/boot/conditionals/OnFSSCondition.kt
package no.nav.boot.conditionals

import no.nav.boot.conditionals.Cluster.Companion.fssClusters
import org.springframework.core.type.AnnotatedTypeMetadata

class OnFSSCondition : OnClusterCondition() {
    override fun clusters(md: AnnotatedTypeMetadata) = fssClusters()
}